.github/workflows/pr-checks.yaml

```diff
@@ -84,8 +84,17 @@ jobs:
 
       - name: Run unittests for policies
         run: |
-         TEST_FILES="./policies ./unittests ./unittests/test_data"
-         /usr/bin/opa test --coverage --format json $TEST_FILES | { T=`mktemp`; tee $T; /usr/bin/opa eval --format pretty --input $T --data hack/simplecov.rego data.simplecov.from_opa > coverage.json; rm -f $T || true; } | jq -j -r 'if .coverage < 100 then "ERROR: Code coverage threshold not met: got \(.coverage) instead of 100.00\n" | halt_error(1) else "" end'
+          TEST_FILES="./policies ./unittests ./unittests/test_data"
+          /usr/bin/opa test --coverage --format json $TEST_FILES \
+            | { T=`mktemp`; \
+              tee $T; \
+             /usr/bin/opa eval \
+              --format pretty \
+              --input $T \
+              --data hack/simplecov.rego \
+              data.simplecov.from_opa > coverage.json; \
+            rm -f $T || true; } \
+          | jq -j -r 'if .coverage < 100 then "ERROR: Code coverage threshold not met: got \(.coverage) instead of 100.00\n" else "" end'| exit ${PIPESTATUS[0]:-0}
```

Comment:
Maybe just specifying `set -o pipefail` may be an elegant solution to this?
https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#set--o-pipefail

Reply:
Yes - much better. Updated to reflect this.